doc/user_guide/electron_holography.rst

```diff
@@ -57,5 +58,38 @@ This option can be used for batch processing as the reconstruction_param can be
 
     >>> wave_image1 = im1.reconstruct_phase(rec_param=reconstruction_param)
 
-Further analyses of the reconstructed wave (phase) can be done using the :py:class:`~._signals.wave_image.ComplexSignal2D` class
-functionality (see dedicated documentation).
\ No newline at end of file
+Further analyses of the reconstructed wave (phase) can be done using :py:class:`~._signals.wave_image.ElectronWaveImage` class
+functionality (see bellow).
+
+
+The ElectronWaveImage class
+===========================
+
+The :py:class:`~._signals.wave_image.ElectronWaveImage` class can hold information about the complex electron
+wave. As such, relevant properties like the `amplitude`, `phase` and the `real` and `imag` part can be
+directly accessed and return appropriate :py:class:`~._signals.signal2d.Signal2D` signals.
+
+To transform a :py:class:`~._signals.complex_signal2d.ComplexSignal2D` (or subclass) into a
+:py:class:`~._signals.wave_image.ElectronWaveImage` use:
+
+.. code-block:: python
+
+    >>> im.set_signal_type('ElectronWaveImage')
```

Comment:
I think that we also decided that there was no need for "_" instead of " " and these are strings with no risk of becoming attributes. So, what about "electron wave"?